R/geom-abline.R

```diff
@@ -73,7 +73,9 @@ geom_abline <- function(mapping = NULL, data = NULL,
                         slope,
                         intercept,
                         na.rm = FALSE,
-                        show.legend = NA) {
+                        show.legend = NA,
+                        stat = "identity",
+                        inherit.aes = FALSE) {
```

Comment:
can you match the argument order with the other geoms. Especially `stat` tend to come earlier